### PR TITLE
[GRIFFIN-367] For task GRIFFIN-367, update local deploy document.

### PR DESCRIPTION
**Pull request by lipzhu**
**What changes were proposed in this pull request?**
1. Remove the diff change history for hive-site.xml.
2. Update the deployment doc for service module package location.

**Does this PR introduce any user-facing change?**
No.

**How was this patch tested?**
Unit Tests

### CODE REVIEW

File: service/pom.xml [hunk 1]
```diff
@@ -385,7 +385,6 @@ under the License.
                 <configuration>
                     <appendAssemblyId>false</appendAssemblyId>
                     <skipAssembly>false</skipAssembly>
-                    <outputDirectory>../target</outputDirectory>
```
Comment (whhe) [inline service/pom.xml:388]: Why move it?  I think it's appropriate to put the deployment package at root directory.

Comment (lipzhu) [reply service/pom.xml:388]: The service-${version}.tar.gz location is not align with the Documention in https://github.com/apache/griffin/blob/master/griffin-doc/deploy/deploy-guide.md, or we can update the documentation to remove the confuse.  

>>>It's easy to build Griffin, just run maven command mvn clean install. Successfully building, you can get service-${version}.tar.gz and measure-${version}.jar from target folder in **service** and **measure** module.

Comment (whhe) [reply service/pom.xml:388]: > The service-${version}.tar.gz location is not align with the Documention in https://github.com/apache/griffin/blob/master/griffin-doc/deploy/deploy-guide.md, or we can update the documentation to remove the confuse.
> 
> > > > It's easy to build Griffin, just run maven command mvn clean install. Successfully building, you can get service-${version}.tar.gz and measure-${version}.jar from target folder in **service** and **measure** module.

You're right. I think it's better to update the doc, and the following commands should also be modified at the same time.

```bash
cd $GRIFFIN_INSTALL_DIR
tar -zxvf service-${version}.tar.gz
```

Comment (chitralverma) [reply service/pom.xml:388]: @lipzhu please update the docs instead

**Review by chitralverma — CHANGES_REQUESTED** [review]

**Review by whhe — APPROVED** [review]

**Pull request merged**